lib/knapsack_pro/runners/queue/rspec_runner.rb

```diff
@@ -107,10 +109,14 @@ def self.run_tests(accumulator)
 
         private
 
+        def self.adapter_class
+          KnapsackPro::Adapters::RSpecAdapter
+        end
+
         def self.log_rspec_command(cli_args, test_file_paths, type)
           case type
           when :subset_queue
-            KnapsackPro.logger.info("To retry the last batch of tests fetched from the API Queue, please run the following command on your machine. (If you use the `-- order random` option, remember to add correct `--seed 123` that you can find at the end of the RSpec output.)")
+            KnapsackPro.logger.info("To retry the last batch of tests fetched from the API Queue, please run the following command on your machine:")
           when :end_of_queue
```

Comment:
This case (invoked on L63) remains unchanged. I didn't like any of the options I came up to make it work. I figured it's better to improve other logs for now, and alter this case if I think of a good-enough solution for it.

Reply:
Does it mean we don't display `--seed 123` in the very last logger message on purpose, right? I checked that, and I don't see the seed:

```
INFO -- : [knapsack_pro] To retry all the tests assigned to this CI node, please run the following command on your machine:
INFO -- : [knapsack_pro] bundle exec rspec --order random --format progress  --default-path spec "spec/pending_spec.rb" "spec/controllers/dashboard/pending_controller_spec.rb"
```

The solution that comes to my mind is to store the used seed on the first call of the `args_with_seed_option_added_when_viable` method. We could store the seed in the Ruby class variable: `@@seed`. The seed is unchanged for each batch of tests fetched from the Queue API, so it's safe to cache it in the class variable. This way, we could display the seed in the last logger message without a need to instantiate the `rspec_runner` variable which you use in L90:

```ruby
# L90
            printable_args = args_with_seed_option_added_when_viable(args, rspec_runner)
            log_rspec_command(printable_args, test_file_paths, :subset_queue)
```

Reply:
Thanks for the idea. I've updated the code to use the class variable.

While I'm not in love with the final solution, I figured it's the best one we came up with _and_ it improves the DX so I guess some trade-offs are justified.